hydrolib/core/dimr/models.py

```diff
@@ -52,7 +52,7 @@ class Component(BaseModel, ABC):
     name: str
     workingDir: Path
     inputFile: Path
-    process: Optional[int]
```

Comment:
This should remain an `int`, right? We only want to ensure that, for example, the integer 3 is written as `<process>0 1 2</process>` and not as `3`?

We still want to be able to do:
```
fm_comp = FMComponent(**data)
fm_comp.process = 4
```

Or did I missunderstand the issue?

Reply:
yes, you should be able to set it like this when using the model, but when it is read from the file it should be the correct format and be "0 1 2 3" when 4 is the input.
I think this should be an union of int and str in that case?
Since the input can be either int or str and the output is always str, since "0 1 2 3" cannot be of type int.